tlvf/yaml/tlvf/ieee_1905_1/eWscAttributes.yaml

```diff
@@ -0,0 +1,39 @@
+#
+---
+_namespace: ieee1905_1
+
+eWscAttributes:
+  _type: enum
+  _enum_storage: uint16_t
+  ATTR_VERSION: 0x104a
+  ATTR_MSG_TYPE: 0x1022
+  ATTR_UUID_E: 0x1047
+  ATTR_UUID_R: 0x1048
+  ATTR_MAC_ADDR: 0x1020
+  ATTR_ENROLLEE_NONCE: 0x101a
+  ATTR_REGISTRAR_NONCE: 0x1039
+  ATTR_PUBLIC_KEY: 0x1032
+  ATTR_AUTH_TYPE_FLAGS: 0x1004
+  ATTR_ENCR_TYPE_FLAGS: 0x1010
+  ATTR_CONN_TYPE_FLAGS: 0x100d
+  ATTR_CONFIG_METHODS: 0x1008
+  ATTR_WSC_STATE: 0x1044
+  ATTR_MANUFACTURER: 0x1021
+  ATTR_MODEL_NAME: 0x1023
+  ATTR_MODEL_NUMBER: 0x1024
+  ATTR_SERIAL_NUMBER: 0x1042
+  ATTR_PRIMARY_DEV_TYPE: 0x1054
+  ATTR_DEV_NAME: 0x1011
+  ATTR_RF_BANDS: 0x103c
+  ATTR_ASSOC_STATE: 0x1002
+  ATTR_DEV_PASSWORD_ID: 0x1012
+  ATTR_CONFIG_ERROR: 0x1009
+  ATTR_OS_VERSION: 0x102d
+  ATTR_VENDOR_EXTENSION: 0x1049
+  ATTR_SSID: 0x1045
+  ATTR_AUTH_TYPE: 0x1003
+  ATTR_ENCR_TYPE: 0x100f
+  ATTR_NETWORK_KEY: 0x1027
+  ATTR_KEY_WRAP_AUTH: 0x101e
+  ATTR_ENCR_SETTINGS: 0x1018
+  ATTR_AUTHENTICATOR: 0x1005
```

Comment:
I will, but Tomer and I didn't fully understand why it's needed

Reply:
It's just whitespace convention, just like the convention not to use \r\n but only \n as newlines.

The only real effect is that if you add a line at the end of the file and there was no newline, you'll get two modified lines in the diff instead of only one.

Also, some editors always add a newline at the end of the file, so if you open a file in such an editor, the last line will be changed even if you have no other modifications.

Finally, when you count the number of lines, some tools will not count that final line without newline as a line, but other tools will. I don't think this is a big deal :-)